src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu.component.tsx

```diff
@@ -41,6 +43,7 @@ export interface ResponsiveVerticalMenuProps extends TagProps {
 const BaseMenu = ({
   children,
   height,
+  launcherButtonAriaLabel = "Main menu",
```

Comment:
question (non-blocking): is `Main menu` an appropriate default here? Not sure who can decide on that though tbh.